python/dgl/sparse/elementwise_op.py

```diff
@@ -45,7 +45,7 @@ def add(
     >>> val = torch.tensor([10, 20, 30])
     >>> A = from_coo(row, col, val)
     >>> B = diag(torch.arange(1, 4))
-    >>> A + B
+    >>> add(A, B)
```

Comment:
Will users then realize they can do `A + B`?

Reply:
This is the doc string for add. Let's me also clarify that this is equivalent to +.